lib/domain/dtos/filters/RunFilterDto.js

```diff
@@ -23,9 +23,16 @@ const DetectorsFilterDto = Joi.object({
     values: Joi.string().trim(),
 });
 
+const EorFilterDto = Joi.object({
+    category: Joi.string(),
+    title: Joi.string(),
+    description: Joi.string(),
+});
+
 exports.RunFilterDto = Joi.object({
     runNumbers: Joi.string().trim(),
     definitions: CustomJoi.stringArray().items(Joi.string().uppercase().trim().valid(...RUN_DEFINITIONS)),
+    eor: EorFilterDto,
```

Comment:
Please avoid too short variables. EOR may mean end of run, which would be the time of the end of run.
Please name it EorReason, everywhere